biogeophys/FatesPlantHydraulicsMod.F90

```diff
@@ -1501,6 +1577,14 @@ subroutine UpdateH2OVeg(nsites,sites,bc_out)
        ! Note that h2oveg_dead is incremented wherever we have litter fluxes
        ! and it will be reduced via an evaporation term
        ! growturn_err is a term to accomodate error in growth or turnover. need to be improved for future(CX) 
+       if (JD_debug) then 
```

Comment:
Also here. Is this likely to be triggered?

Reply:
several writes converted to debug, several removed